modules/formulize/class/elementrenderer.php

```diff
@@ -1042,7 +1042,7 @@ function constructElement($form_ele_id, $ele_value, $entry, $isDisabled=false, $
 			break;
 
 
-			case 'upload':
+			case 'fileUpload':
```

Comment:
I checked the file upload element object type is XoopsFormFile extends icms_form_elements_file, not formulizeFileUploadElement.